src/App.js

```diff
@@ -2,12 +2,24 @@ import './App.css';
 
 import Header from './Header';
 import Home from './Home';
+import Checkout from './Checkout';
```

Comment:
Make different folders for Components and Pages. Also use barrel export for them